File: KargoPaylasimJava/app/src/main/java/com/example/kargopaylasimjava/dto/ReceiverDtos.java
package com.example.kargopaylasimjava.dto;

public class ReceiverDtos {

    public static class ReceiverLookupReq {
        public String phone;
        public ReceiverLookupReq(String phone) { this.phone = phone; }
    }

    public static class ReceiverLookupResp {
        public int receiver_user_id;
        public String masked_first_name;
        public String masked_last_name;
    }
}

File: KargoPaylasimKotlin/app/src/main/java/com/example/kargopaylasimkotlin/dto/ReceiverDtos.kt
package com.example.kargopaylasimkotlin.dto

data class ReceiverLookupReq(val phone: String)

data class ReceiverLookupResp(
    val receiver_user_id: Int,
    val masked_first_name: String,
    val masked_last_name: String
)
